noxfile.py

```diff
@@ -48,6 +48,14 @@ def default(session):
     else:
         session.install("ipython")
 
+    # opentelemetry was not added to [all] because opentelemetry does not support Python 2.
+    # Exporter does not need to be in nox thus it has been added to README documentation
+    if session.python != "2.7":
+        session.install(
```

Comment:
It's important to include `opentelemetry` somewhere in `setup.py` so that the dependency resolver can check if a compatible version is installed.

```suggestion
        session.install("-e", ".[opentelemetry]")
```